app/scripts/views/mixins/password-mixin.js

```diff
@@ -39,17 +32,10 @@ define(function (require, exports, module) {
       try {
         var passwordField = this.$('.password');
         if (isVisible) {
-          passwordField.attr('type', 'text').attr('autocomplete', 'off')
-            .attr('autocorrect', 'off').attr('autocapitalize', 'off');
```

Comment:
@vladikoff adding back all the changes below will include the deleted function isPasswordAutoCompleteDisabled()
'else {
          passwordField.attr('type', 'password');
          if (this.isPasswordAutoCompleteDisabled()) {
            passwordField.attr('autocomplete', 'off');
          } else { '